### PR TITLE
update tomcat test.properties and test.sh

### PR DESCRIPTION
**Pull request by yinggzz**
related to issue #3398

### CODE REVIEW

**Comment by sophia-guo** [comment]
You need to update the code mentioned in https://github.com/adoptium/aqa-tests/issues/3398#issuecomment-1081120600 to fit in your environment. For example in our environment there is no $HOME, which caused the failure 
Screen Shot 2022-03-29 at 12 11 30 PM

**Comment by yinggzz** [comment]
new grinder link: https://ci.adoptopenjdk.net/job/Grinder/4173/console
the error still persists:
```
19:05:51      [junit] 	org.apache.catalina.LifecycleException: The configured protocol [org.apache.coyote.http11.Http11AprProtocol] requires the APR/native library which is not available
```

**Comment by sophia-guo** [comment]
@yinggzz did you still get the error mentioned in https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082334188. The error mentioned https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082588973 is different  from https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082334188, though you hit it before. The grinder output is too large to load, I can't see the real reason of the failure. Could you rebase your branch and run again? We have two PRs in to help reducing the output.

The error in https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082334188 means the building error of tomcat-native. And error in https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082588973 relates with tests setting, i.e. test native path, which we didn't set it up before. So better to rerun with rebased branch and see if we can load the full console output.

**Comment by yinggzz** [comment]
@sophia-guo I am not getting the error in [#3507 (comment)](https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082334188) anymorem however, I still get error in [#3507 (comment)](https://github.com/adoptium/aqa-tests/pull/3507#issuecomment-1082588973) relating to test setting.

New grinder link: https://ci.adoptopenjdk.net/job/Grinder/4204/console

error: 
```
14:21:08      [junit] 30-Mar-2022 18:21:07.965 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent The Apache Tomcat Native library which allows using OpenSSL was not found on the java.library.path: [/tomcat/output/build/bin]
14:21:09      [junit] 30-Mar-2022 18:21:09.037 SEVERE [main] org.apache.catalina.util.LifecycleBase.handleSubClassException Failed to initialize component [Connector[org.apache.coyote.http11.Http11AprProtocol-auto-1]]
14:21:09      [junit] 	org.apache.catalina.LifecycleException: The configured protocol [org.apache.coyote.http11.Http11AprProtocol] requires the APR/native library which is not available
…
14:21:26  BUILD FAILED
14:21:26  /tomcat/build.xml:1903: Some tests completed with an Error. See /tomcat/output/build/logs for details, search for "ERROR".
14:21:26  
14:21:26  Total time: 183 minutes 51 seconds
14:21:27  
14:21:27  tomcat_test_0_FAILED
```

**Comment by yinggzz** [comment]
new test ran after adding the lines 74-78 from travis.yml: https://ci.adoptopenjdk.net/job/Grinder/4275/console

error is still:
```
23:12:44      [junit] 04-Apr-2022 03:12:43.824 SEVERE [main] org.apache.catalina.util.LifecycleBase.handleSubClassException Failed to initialize component [Connector[org.apache.coyote.http11.Http11AprProtocol-auto-1]]
23:12:44      [junit] 	org.apache.catalina.LifecycleException: The configured protocol [org.apache.coyote.http11.Http11AprProtocol] requires the APR/native library which is not available
```

File: external/tomcat/test.sh [hunk 1]
```diff
@@ -16,6 +16,30 @@ source $(dirname "$0")/test_base_functions.sh
...
+echo >> build.properties
+echo "test.threads=4" >> build.properties
+echo "test.relaxTiming=true" >> build.properties
+echo "test.excludePerformance=true" >> build.properties
+echo "test.openssl.path=/dev/null/openssl" >> build.properties
+echo "test.apr.loc=$CURRENT/tmp/tomcat-native-build/lib" >> build.properties
+
 set -e
 echo "Building tomcat" && \
 cp build.properties.default build.properties && \
```
Comment (sophia-guo) [inline external/tomcat/test.sh:45]: The line should be removed as we have copied and updated build.properties file. This line might override what we have done.

**Comment by sophia-guo** [comment]
Would also suggest using existing environment variable TEST_HOME as the HOME environment variable  instead of `current`, which is not related with the failure. Just don't want to create unnecessary duplicate variable.

File: external/tomcat/test.sh (hunk 1 above)
Comment (yinggzz) [reply external/tomcat/test.sh]: Line 45 removed, thank you!

**Comment by yinggzz** [comment]
new grinder link: https://ci.adoptopenjdk.net/job/Grinder/4416/console

**Review by sophia-guo — APPROVED** [review]

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**